testingScenarios

```diff
@@ -165,3 +165,74 @@ SCENARIO("game of mixed spare after strike"){
         }
     }
 }
+
+SCENARIO("game have ten turns"){
+    GIVEN("counter of turns") {
+        int turnCounter = 0;
+        int maxTurns = 10;
+        WHEN ("turnCounter == maxTurns") {
```

Comment:
shouldn't we first start some method before we check that condition?
in this case for example playGame()

Reply:
I think so too